docs/installation/configuration.md

```diff
@@ -185,3 +185,17 @@ This results in a server with:
 - Flannel backend set to `none`
 - The token set to `secret`
 - Debug logging enabled
+
+### Kubelet configuration
+
+:::info Version Gate
+Due to an [upstream bug](https://github.com/kubernetes/kubernetes/pull/127421) which only gets fixed in the v1.32 minor, the drop-in directory for kubelet configuration files or the config file (options 1 and 2 below) are only available in v1.32 and above. For lower minors, you should use the kubelet args directly (option number 3 below)
+:::
+
+Following on from upstream behavior, kubelet configuration can be changed in different ways with a specific [order of precedence](https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/#kubelet-configuration-merging-order). 
+
+K3s uses a default kubelet configuration which is stored under `/var/lib/rancher/k3s/agent/etc/kubelet.conf.d/00-k3s-defaults.conf`. If you would like to change the default configuration parameters, there are three ways to do so:
```

Comment:
The recommended way is to add a user drop-in to `/var/lib/rancher/k3s/agent/etc/kubelet.conf.d/`. Specifying a config file or config dir paths via kubelet-arg will work but does not integrate as well as simply dropping a file in the config-dir that we manage.